### DISCUSSION TITLE
Real A320 pilots are getting excited.

### DISCUSSION THREAD

**Discussion by Fujix1** [flybywiresim/aircraft#3137] [General]
Two real A320neo pilots introduced the mod, they are flying togher and getting super exicted.

1) Captain Joe's channel: https://www.youtube.com/watch?v=VsxKZUElIis
2) Captain Ryu's channel: https://www.youtube.com/watch?v=75QUscaWn_o

During the flight, they pointed out some problems. I pick up :

- 1st movie at 11:13 - The unit is not lb.
- 1st movie at 12:02 - ALTN FUEL doesn't work.
- 1st movie at 13:11 - SEC F PLAN doessn't work.
- 1st moive at 14:58 - SIRAO is not F390.
- 1st movie at 18:47 - It's not lb.
- 2nd movie at 6:25 - GEN1 should not be OFF.
- 2nd movie at 7:09 - No hydro screen yet.
- 2nd movie at 11:24 - Doesn't accept CLR HOLD. Have to press IMM EXIT to do that.
- 2nd movie at 12:56 - HOLD is not reflected.
- 2nd moive at 14:34 - Autopilot does't work?

Although, they highly appreciate the mod.

**Comment by kiokio0987** [comment]
I could tell from the atmosphere that they were very happy.
